### ISSUE TITLE
jenv does not support path that contains a space

### ISSUE THREAD

**Issue by Atry** [gcuisinier/jenv#160]
```
$ jenv add '/Library/Internet Plug-Ins/JavaAppletPlugin.plugin/Contents/Home'
/usr/local/Cellar/jenv/0.4.4/libexec/libexec/jenv-add: line 51: cd: /Library/Internet: No such file or directory
```

**Comment by pparavicini** [comment]
Have you tried escaping the space after 'Internet' as in:

$ jenv add '/Library/Internet\ Plug-Ins/[...]'  ?

**Comment by Atry** [comment]
`'/Library/Internet\ Plug-Ins/[...]'` is a wrong argument, as single quotes can contain spaces already.

**Comment by neuromancer-es** [comment]
I am facing this issue. I fixed it and want to share.

I found the code had $JENV_JAVAPATH quoted and unquoted thoughout.
And fix #161 was not implemented as the first cd $JENV_JAVAPATH was unquoted
I quoted every $JENV_JAVAPATH in the file  and it finally worked. No other way to make it work.
I'm happy to share more info. Excuse me if this isn't the right place to post

JENV VERSION:
jenv: stable 0.4.4, HEAD
Manage your Java environment
https://www.jenv.be/
/usr/local/Cellar/jenv/0.4.4 (78 files, 66KB) *
  Built from source on 2018-08-18 at 18:53:31
